Mathlib/Data/Finsupp/Lex.lean

```diff
@@ -82,7 +82,8 @@ instance Lex.partialOrder [PartialOrder N] : PartialOrder (Lex (α →₀ N)) wh
 
 /-- The linear order on `Finsupp`s obtained by the lexicographic ordering. -/
 instance Lex.linearOrder [LinearOrder N] : LinearOrder (Lex (α →₀ N)) where
-  __ := Lex.partialOrder (α := α) (N := N)
+  lt := (· < ·)
+  le := (· ≤ ·)
```

Comment:
Hmmm, not sure why I don't get the correct syntactic form with `__ := Lex.partialOrder (α := α) (N := N)` ...